.github/workflows/testing.yml

```diff
@@ -5,43 +5,32 @@ on:
     branches:
     - main
     - stable
+    - v*.*.x
   pull_request:
   workflow_dispatch:
 
 jobs:
 
   initial-tests:
     name: ${{ matrix.name }}
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
```

Comment:
The old version of this line meant that the tests were only run on Linux, and not on mac or Windows.  Oops!

Reply:
Oh, understood!